config/user.yaml

```diff
@@ -5,5 +5,6 @@ email:
   port: 1025
   ssl: false
   username: None
-model_language: english
+language: english
+model_version: 0.0.1
```

Comment:
If version 0.0.1 is specified, it will use the oldest version of all models. Models don't necessarily share the same version, as they are incremented on each update to keep version control intact. Do we need to specify versions for the models at all?

Reply:
Sorry, the naming is very confusing. That variable is meant to be a release version of the language specified, so that we can make releases for the languages were we specify which models and versions to use for every automation module. Then the user can easily switch between the releases of the language and also download different releases and languages. 

Having releases is not necessary, but I think it is a nice feature to have as a user and developer.

Will rename the variable to `language_version`.

Reply:
Ok I understand then. Thanks for clarifying :)